test/Microsoft.AspNetCore.Razor.Tools.Test/Infrastructure/ToolTestFixture.cs

```diff
@@ -0,0 +1,124 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Xml.Linq;
+using Microsoft.DotNet.Cli.Utils;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Razor.Tools.Test.Infrastructure
+{
+    public class ToolTestFixture : IDisposable
+    {
+        private readonly string _defaultNugetPackageLocation;
+
+        public ToolTestFixture()
+        {
+            PackSrcDirectory();
+
+            _defaultNugetPackageLocation = Environment.GetEnvironmentVariable("NUGET_PACKAGES");
+            if (string.IsNullOrWhiteSpace(_defaultNugetPackageLocation))
+            {
+                _defaultNugetPackageLocation = Environment.ExpandEnvironmentVariables(
+                    @"%USERPROFILE%\.nuget\packages");
```

Comment:
%USERPROFILES% isn't defined in xplat. Clearing it i.e. using the value of `Environment.GetEnvironmentVariable("NUGET_PACKAGES")`) all the time, should suffice.